docs/src/components/Tile.tsx

```diff
@@ -209,7 +209,7 @@ export default function Tile({
             <TileTitle>{title}</TileTitle>
             <div
               css={css`
-                margin: 0.5rem 0 ${href ? "1.5rem" : "0"};
+                margin: 0.5rem 0 ${href ? (linkContent && "1.5rem") || "0.5rem" : "0"};
```

Comment:
No reason to add padding just because it's a link. Also, at the bottom add `StyledEndLinkWrapper` if both `href` and `linkContent` are provided, not only `href`, otherwise there will be empty elements.
```suggestion
                margin: 0.5rem 0 ${href && linkContent ? "1.5rem" : "0"};
```